README.md

```diff
@@ -1 +1,111 @@
-# other-hash-methods-code-along
+# Higher Level Hash Methods
+
+##Objectives
+* Learn some interesting methods we can call on hashes
+
+-
+
+Previously, you were asked to iterate over a hash and collect the key that pointed to the lowest value. We asked you not to use some of the higher level hash methods there. Now, we're going to learn a few tricks that can make a task like that much easier. 
+
+### `.values`
+
+You can collect all of the values in a hash with the `.values` method: 
+
+```ruby
+family_members = {mom: "Victoria", dad: "Richard", sister: "Zoe"}
+
+family_members.values
+#  => ["Victoria", "Richard", "Zoe"]
+```
+
+We can see that the `.values` method returns an array of the values of the keys in the hash. 
+
+### `.keys`
+
+This method, not surprisingly, returns an array containing all of the keys in the hash that `.keys` has been called on: 
+
+```ruby
+family_members = {mom: "Victoria", dad: "Richard", sister: "Zoe"}
+
+family_members.keys
+#  => [:mom, :dad, :sister]
+```
+
+### `.min`
+
+You can use the `.min` method on a hash to return the key/value *pair* that contains that **lowest** value: 
+
+```ruby
+food_items = {apples: 45, pears: 12}
+
+food_items.min
+#  => [:apples, 45] 
+```
+
+These are only a few of the many helpful methods out there. Be there to check out the [Ruby Docs on Hashes](http://ruby-doc.org/core-2.2.2/Hash.html) to learn more. 
+
+Let's practice before you move on to the next challenge: 
+
+
+## Code Along: Manipulating Nested Hashes
+
+You'll be coding your solution to this challenge in `lib/values.rb`, `lib/keys.rb`, and `lib/min.rb`. We have a nested hash of grocery item.
+
+```ruby
+groceries = {
+  dairy: ["milk", "yogurt", "cheese"],
+  vegetable: ["carrots", "broccoli", "cucumbers"],
+  meat: ["chicken", "steak", "salmon"],
+  grains: ["rice", "pasta"]
+}
+```
+ 
+## Challenge 1: Return the values
+Use the `.values` method to collect all of the values of the grocery type keys (`:dairy`, `:vegetables`, `:meat`, `:grains`). The method should return a *one-dimensional* (or "flat") array that *only* includes the values (groceries such as "milk" and "carrots") without their keys. 
```

Comment:
I would argue that we should keep this bc it's right in the middle of the nested hash section.